src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,125 +1,184 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Locale;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
+    private static final String DATE_PATTERN_SPECIFIC_TIME_ZONE = "yyyy-MM-dd'T'HH:mm:ss'Z'";
+    private static final String TIMEZONE_UTC = "UTC";
+    private static final String ZONE_OFFSET = "+02:00";
+    private static final String CUSTOM_PATTERN = "d MMM yyyy";
+    private static final String FORMAT_PATTERN = "dd MMMM yyyy HH:mm";
```

Comment:
```suggestion
    private static final String DATE_PATTERN = "d MMM yyyy";
    private static final String DATE_TIME_PATTERN = "dd MMMM yyyy HH:mm";
```

Reply:
I gave thees names acording to my methods names - Ok I have renamed